src/transformer/encode.ts

```diff
@@ -76,6 +77,15 @@ const footnoteContents = (node: ManuscriptNode): string => {
   return serializeToXML(output)
 }
 
+const keywordContents = (node: ManuscriptNode): string => {
+  const output = serializer.serializeNode(node) as HTMLElement
+  output.querySelectorAll('span').forEach((element) => {
```

Comment:
Why do we serialize an entire node without attributes and not just string of content? Also will it not get wrapped eventually into more spans (with encode/decoding happening multiple times). E.g.: <span><span><span>some text</span></span></span> ?

Reply:
Fixed.